tests/zzz_copy_sut_archive_files.rb

```diff
@@ -1,16 +1,15 @@
 require 'fileutils'
 
 step "Copy archive files from SUT" do
-  archive_files = ENV['SUT_ARCHIVE_FILES'].split(",")
-  job_name = ENV['PUPPET_GATLING_SIMULATION_ID']
-  Beaker::Log.notify("Copying #{archive_files.count} archive files from SUT")
-
-  archive_dir = "../puppet-gatling/#{job_name}/sut_archive_files"
-
-  FileUtils.mkdir_p(archive_dir)
-
-  archive_files.each do |s|
-    Beaker::Log.notify("Copying archive file '#{s}'")
-    scp_from(master, s, archive_dir)
-  end
+  today = Date.today().to_s
+  # truncate the job name so it only has the name-y part and no parameters
+  job_name = (ENV['JOB_NAME'] || 'unknown_jenkins_job')
+                 .sub(/[A-Z0-9_]+=.*$/, '')
+                 .gsub(/[\/,.]/, '_')[0..200]
+  archive_name = "#{job_name}__#{ENV['BUILD_ID']}__#{today}__perf-files.tgz"
```

Comment:
If outside of jenkins this name will be the same all day long.

Reply:
Yes.  This is primarily for use in CI, although it could be handy when running locally just because it will copy all of the results you care about into one place.  This is also completely stolen from pe_acceptance_tests :-)

Reply:
Also, you still get per node directories as well, so your results per run will be preserved unless you do it on the same set of nodes.

Reply:
I was thinking same set of nodes... like multiple dev runs trying out different tweaks, then wanting to compare the results of each to look for performance changes.